polaris-tokens/package.json

```
@@ -26,7 +26,6 @@
     "dev": "rollup -c -w",
     "lint": "run-p lint:*",
     "lint:js": "TIMING=1 eslint --cache .",
-    "lint:styles": "stylelint '**/*.{css,scss}'",
```

This was failing with the error:

```
Error: No files matching the pattern "**/*.{css,scss}" were found.
```

Which makes sense because there's no `.css` or `.scss` files in this folder.

But why was it working before?

Because the previous CI task would (unnecessarily) run `yarn build` which generates `dist/**/*.css` files. So, when the glob was run, there _are_ files for `stylelint` to match against, and hence no error. _But_ those files are all in `dist/` which [is ignored in `stylelintrc.js`](https://github.com/Shopify/polaris/blob/9c24a465c5e001e148bde335bf6319e924f4b1d6/.stylelintrc.js#L43), so `stylelint` would silently ignore all the files and exit successfully having done nothing.